docs/contributing/commit-discipline.md

```diff
@@ -1,7 +1,3 @@
-# Version control
-
-## Commit discipline
```

Comment:
Sphinx uses the first header as the page title, so this page should begin with `# Commit discipline`. Without this, it’s taking “Commit messages” as the page title (as you can see in the [preview build for this PR](https://zulip--20459.org.readthedocs.build/en/20459/contributing/commit-discipline.html)), even though that’s a section further down within the page.

Reply:
Thanks, I did suspect there was a reason why every other page has a top level title. I've updated.